src/transformers/models/data2vec/convert_data2vec_audio_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -0,0 +1,246 @@
+# coding=utf-8
```

Comment:
To run the checkpoint conversion script:
```
python -m transformers.models.data2vec.convert_data2vec_audio_original_pytorch_checkpoint_to_pytorch --checkpoint_path <your_path_to>/audio_base_ls.pt --pytorch_dump_folder_path <your_output_path> --not_finetuned
```
`audio_base_ls.pt` can be downloaded from [here](https://dl.fbaipublicfiles.com/fairseq/data2vec/audio_base_ls.pt).  
You will see two print statements showing that the forward pass matches up until the addition of convolutional positional embeddings.